components/blocks/cards/LogInCard/LogInCard.vue

```diff
@@ -51,6 +52,7 @@ async function login() {
         state.email.value = ''
         state.password.value = ''
         state.showPassword.value = false
+        emit('close')
```

Comment:
I'm a bit confused here. It seemed like it was still closing even before this line was added.

Reply:
I'm confused why it shows that there? because thats not part of my local

Reply:
@erunks why was this added?